.github/workflows/linux.yml

```diff
@@ -114,8 +114,15 @@ jobs:
           name: "arturo-${{matrix.version}}-${{matrix.arch}}-linux"
           path: bin/arturo
 
-      - name: Run unit tests
-        if: matrix.arch == 'amd64' && (matrix.version == 'full' || matrix.version == 'mini')
+      - name: Run unit tests (mini)
+        if: matrix.arch == 'amd64' && (matrix.version == 'mini')
         run: |
           echo "${{ github.workspace }}/bin" >> $GITHUB_PATH
-          arturo tools/tester.art
\ No newline at end of file
+          arturo tools/tester.art
+      
+      - name: Run unit tests (full)
+        if: matrix.arch == 'amd64' && (matrix.version == 'full')
+        run: |
+          echo "${{ github.workspace }}/bin" >> $GITHUB_PATH
+          ./bin/arturo --package install unitt 1.1.2
```

Comment:
```suggestion
```

As I say in the other comment (in `unitt-tester.art`), we don't really need that. 😉 

P.S. Obviously, it's not that important, but the less lines of code we have, the less we have to debug...